packages/early-boot-config/early-boot-config.spec

```diff
@@ -87,66 +73,91 @@ Requires: %{_cross_os}early-boot-config-local
     -p vmware-guestinfo-user-data-provider
 %endif
 
+%global cargo_outdir %{getenv:HOME}/.cache/%{__cargo_target}/release
```

Comment:
As long as we're using `%cargo_build` then this will be the right path, since we set the target directory in the environment.

I opened https://github.com/bottlerocket-os/bottlerocket-sdk/issues/159 to add standard versions of these macros.